src/utils/common.ts

```diff
@@ -38,3 +44,8 @@ export function shareToTwitter(text: string) {
   window.open(`https://twitter.com/intent/tweet?text=${encodeURIComponent(text)}`, "_blank");
 }
 
+const DEFAULT_DECIMALS = 2
+
+export const formatNumberWithCommas = (value: number, decimals: number = DEFAULT_DECIMALS) =>
+  value !== undefined ? parseFloat(value.toFixed(decimals)).toLocaleString('en-US') : undefined
+
```

Comment:
there is also a very similiar function to this already with `formatAmount`

Reply:
doesn't that one only work for BigNumber inputs though